src/Raven.Server/Documents/Handlers/RevisionsHandler.cs

```diff
@@ -180,6 +181,24 @@ public async Task GetRevisionsFor()
         public async Task Revert()
         {
             RevertRevisionsRequest configuration;
+            var collection = GetQueryOptionalStringValue("collection")?.Trim();
```

Comment:
You want to add this to the 'RevertRevisionsRequest' as `string[] Collections { get; set; }`

Reply:
Done